module/connection/ServerDialog.kt

```diff
@@ -197,15 +203,27 @@ object ServerDialog {
     }
 
     @Composable
-    private fun CACertificateFormField(state: ConnectServerForm) {
+    private fun CACertificateFormField(state: ConnectServerForm, window: ComposeDialog) {
         Field(label = Label.CA_CERTIFICATE) {
-            TextInput(
-                value = state.caCertificate,
-                placeholder = "${Label.PATH_TO_CA_CERTIFICATE} (${Label.OPTIONAL.lowercase()})",
-                onValueChange = { state.caCertificate = it },
-                enabled = StudioState.client.isDisconnected,
-                modifier = Modifier.fillMaxSize(),
-            )
+            Row {
+                TextInput(
+                    value = state.caCertificate,
+                    placeholder = "${Label.PATH_TO_CA_CERTIFICATE} (${Label.OPTIONAL.lowercase()})",
+                    onValueChange = { state.caCertificate = it },
+                    enabled = StudioState.client.isDisconnected,
+                    modifier = Modifier.weight(1f)
+                )
+                RowSpacer()
+                IconButton(
+                    icon = Icon.FOLDER_OPEN,
+                    tooltip = Tooltip.Arg(Label.SELECT_CERTIFICATE_FILE)
+                ) {
+                    val certSelectionResult = selectFilePath(window, Label.SELECT_CERTIFICATE_FILE, SelectorOptions.FILES_ONLY)
```

Comment:
`certSelectionResult` sounds like it returns more than just a path - perhaps some kind of result code, which is not true here:
```suggestion
                    val selectedPath = selectFilePath(window, Label.SELECT_CERTIFICATE_FILE, SelectorOptions.FILES_ONLY)
```

Reply:
Done, went with `selectedFilePath`.